test/views/task_list_view_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule CodeCorps.TaskListViewTest do
+  use CodeCorps.ConnCase, async: true
+
+  import Phoenix.View, only: [render: 3]
+
+  test "renders all attributes and relationships properly" do
+    project = insert(:project)
+    task_list_a = insert(:task_list, rank: 1000, project: project)
+    task_list_b = insert(:task_list, rank: 500, project: project)
+    task = insert(:task, rank: 1000, task_list: task_list_a)
+
+    rendered_json =  render(CodeCorps.TaskListView, "show.json-api", data: [task_list_a, task_list_b])
```

Comment:
Good catch. I was working on seeing if I could work with `position` in the return and needed multiple items to verify. Since that's not how this works (`rank` is returned) we can go back to just one.